openage/convert/main.py

```diff
@@ -159,25 +171,45 @@ def acquire_conversion_source_dir():
 
     if 'AGE2DIR' in os.environ:
         sourcedir = os.environ['AGE2DIR']
-        print("using AGE2DIR: " + sourcedir)
+        print("found environment variable AGE2DIR")
```

Comment:
"Using" is printed below on line 198, for every path (after conversion from relative to absolute paths & expansion of variables)